casual/casual-api/src/main/java/se/laz/casual/config/Startup.java

```diff
@@ -101,6 +103,7 @@ public Builder withServices( List<String> services )
 
         public Startup build()
         {
+            mode = (null == mode) ? Mode.fromName(Optional.ofNullable(System.getenv(CASUAL_INBOUND_STARTUP_MODE_ENV_NAME)).orElseGet(Mode.IMMEDIATE::getName)) : mode;
```

Comment:
We already have ConfigurationService that can and should serve this config. I can see though that the code calling this from Inbound is some "older" code not touched by this PR. Also, wouldn't this fail to read the inbound startup mode if it is configured by file?

Reply:
Fixed, going with config files and also testing using config files and overriding the inbound startup mode via env var.